.clang-format

```diff
@@ -1 +1,7 @@
-BasedOnStyle: LLVM
\ No newline at end of file
+BasedOnStyle: LLVM
+IndentWidth: 2
+TabWidth: 2
+AccessModifierOffset: -2
+UseTab: Never
+ColumnLimit: 80
```

Comment:
80 is way too small. Let it be at least 120 or even 160.

Reply:
Solved by #165